src/main/java/org/fungover/haze/HazeDatabase.java

```diff
@@ -1,5 +1,6 @@
 package org.fungover.haze;
 
+import java.util.ArrayList;
```

Comment:
Is this used somewhere or is it a leftover since code was moved from here to HazeList?